packages/components/scripts/generate-icons.js

```diff
@@ -0,0 +1,20 @@
+const chalk = require("chalk");
+const { clearDir } = require("../../../scripts/helper");
+const { optimize } = require("../../icons/scripts/optimize");
+const { generateSvgComponents, generateIndexFile } = require("./generate-react-components");
+
+const SRC_PATH = "../icons/src";
+const GENERATED_ICON_PATH = "./src/icons/src/generated-icons";
+
+console.log("\nBuild started...");
+
+optimize(SRC_PATH)
```

Comment:
Something weird with the naming here, I would have expect this code to start by calling something call "optimized" I would have expect something retrieving all the svg files.

Maybe `optimize` is having too many concerns?